tests/integration/integration_test.go

```diff
@@ -82,7 +82,7 @@ func TestStackTagValidation(t *testing.T) {
 
 		stdout, stderr := e.RunCommandExpectError("pulumi", "stack", "init", "invalid name (spaces, parens, etc.)")
 		assert.Equal(t, "", stdout)
-		assert.Contains(t, stderr, "stack names may only contain alphanumeric, hyphens, underscores, or periods")
+		assert.Contains(t, stderr, "stack names are limited to 100 characters and may only contain alphanumeric, hyphens, underscores, or periods")
```

Comment:
theres a lint error about line-length, this should fix it up

```suggestion
		assert.Contains(t, stderr,
			"stack names are limited to 100 characters and may only contain alphanumeric, hyphens, underscores, or periods")
```

Reply:
thanks ✌️